README.md

```diff
@@ -1,15 +1,24 @@
 # Scaffold for social media app with Ruby on Rails
 
-> This repo includes intial code for social media app with basic styling. Its purpose is to be a starting point for Microverse students.
+> This purpose of this project is to practice all skills and techniques learnt during the course of Rails to build this social media application. And in this project, features like, creating posts, liking a post, adding friends were implemented.
```

Comment:
- [ ] The should say `The purpose...` instead of `This purpose...` 

- [ ] change `learnt` for `learned`